src/store/ducks/tests/fixtures.ts

```diff
@@ -317,8 +317,8 @@ export const sortedPlanRecordArray = [
     plan_date: '2019-05-24',
     plan_effective_period_end: '2019-09-30',
     plan_effective_period_start: '2019-05-27',
-    plan_fi_reason: '',
-    plan_fi_status: '',
+    plan_fi_reason: ' NULL',
```

Comment:
Why are we doing this? 

- is it possible to get plans with the Fi reason and status as `''`?
- is it possible to get plans with the Fi reason and status as `' NULL'`?

Reply:
seems like we need to cleanup the fixtures data. @moshthepitt i do not know if there could be cases where fi_plan_reason: and plan_fi_status is ' NULL'. If this is not a possible case , I could remove all plans in fixtures that has the named fields as ' NULL'.